### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by JuantAldea**
Standard firmware.bin takes 60668 bytes

### CODE REVIEW

File: frequencies.c [hunk 1]
```diff
@@ -61,40 +62,41 @@ const freq_band_table_t frequencyBandTable[] =
 #endif
 
 
+
 const uint16_t gStepFrequencyTable[] = {
-	250, 500, 625, 1000, 1250, 2500, 833,
```
Comment (egzumer) [inline frequencies.c:65]: This seems to break backwards compatibility. It was over engineered like that for a reason. Only 250, 500, 625, 1000, 1250, 2500, 833 are standard steps, these have to be 0-6.

Comment (JuantAldea) [reply frequencies.c:65]: Yeah... an option could be to store the actual step value and "find" the ID of the closest step that is available in the currently installed firmware.

Comment (egzumer) [reply frequencies.c:65]: I removed changes to gStepFrequencyTable

**Comment by egzumer** [comment]
merged with exception of gStepFrequencyTable

Thanks.

**Comment by JuantAldea** [comment]
My pleasure!

**Pull request closed without merging**